client/gatsby-browser.js

```diff
@@ -13,6 +13,9 @@ import "videojs-record/dist/css/videojs.record.css";
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import Record from "videojs-record/dist/videojs.record.js";
 import "styles/layout.css";
+import { loadSentry } from "./src/helpers";
+
+loadSentry();
```

Comment:
is it disabled when developing locally?

Reply:
[8ab6a80](https://github.com/mentorpal/mentor-admin/pull/190/commits/8ab6a80d8031b2baf19c39fe1ed81595cf5beaa1)